lib/dispatch.rb

```diff
@@ -0,0 +1,24 @@
+require 'twilio-ruby'
+require 'dotenv'
+
+class Dispatch
+Dotenv.load
+
+  def send_sms
+
+  id = ENV["AUTH_ID"]
+  pw = ENV["AUTH_TOKEN"]
+  customers = { ENV["NUMBER1"] => "insert_name_here"}
+
+  @client = Twilio::REST::Client.new id, pw
```

Comment:
Would have been better to use dependency injection here, to no couple your Dispatch class to the Twilio client